### ISSUE TITLE
Request body larger than maxBodyLength limit

### ISSUE THREAD

**Issue by smartprogrammer93** [jsreport/jsreport-dotnet#55]
I am getting this:
``` 
jsreport.Local.JsReportBinaryException: 'Error rendering report: rendering has finished with errors:
A critical error occurred while trying to execute the render command (2).
 Error while executing request to remote server.
 Request body larger than maxBodyLength limit (1).
 caused by error (2):-> 
stackError:    at onCriticalError (D:\snapshot\jsreport\node_modules\jsreport-cli\lib\commands\render.js:302:19)
at D:\snapshot\jsreport\node_modules\jsreport-cli\lib\commands\render.js:201:16
at processTicksAndRejections (internal/process/task_queues.js:97:5)caused by error (1):'
``` 
Although i did set the limit to 100mb like this:
``` 
services.AddJsReport(new LocalReporting()
               .UseBinary(JsReportBinary.GetBinary())
               .Configure(cfg =>
               {
                   var expressConfig =  new ExpressConfiguration();
                   expressConfig.InputRequestLimit = "100mb";
                   cfg.Extensions.Express = expressConfig;
                   return cfg.AllowedLocalFilesAccess()
                           .BaseUrl(env.WebRootPath);
               })
               .KillRunningJsReportProcesses()
               .AsUtility()
               .Create());
```

**Comment by pofider** [comment]
This should be fixed in the later [jsreport.Binary nuget](https://www.nuget.org/packages/jsreport.Binary/).
What version do you use now?

**Comment by smartprogrammer93** [comment]
You are right,

I was using version 2.9
solved after updating to 2.11

Thanks a lot
